src/pages/NewChatPage.js

```diff
@@ -108,17 +108,21 @@ class NewChatPage extends Component {
             }
         }
 
```

Comment:
Actually, could we also get a comment added here that goes like:

> Filtering out selected users from the search results

Same in src/pages/workspace/WorkspaceInvitePage.js

Reply:
Sure.

Reply:
Done